src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -356,7 +362,8 @@ public boolean onCommand(@Nullable CommandSender sender, @Nullable Command comma
 			} else if (args[0].equalsIgnoreCase("gen-docs")) {
 				File templateDir = new File(Skript.getInstance().getDataFolder() + "/doc-templates/");
 				if (!templateDir.exists()) {
-					Skript.info(sender, "Documentation templates not found. Cannot generate docs!");
+					Skript.error(sender, "Documentation templates not found. Cannot generate docs!");
```

Comment:
Can you make it copy the `docs` folder contens from the source/jar to `plugins/Skript/doc-templates`

Reply:
No because you can't copy directories from the resource.

Reply:
Once you add it once to the test runner folder, you don't have to do it again.

Reply:
The idea of having a gradle task for this is to generate the docs automatically with one click and no need to copy/paste anything manually and since the templates may change from time to time it wouldn't be ideal this way, I can do that in my PR if you want (add a check in `/sk gen-docs` if is SkriptTest.GEN_DOCS then copy the templates from the jar to Skript folder)

Reply:
You can't without hard coding every file. You have to know every item in the docs folder, you can't transfer directories as an input stream is a single file.

Reply:
You can make the gradle task do it, doesn't have to use the resource system Skript has built in. Otherwise, you can extend that resource system to directories

Reply:
I think it might be possible if we insert a .TXT file into the jar with a list of the file paths generated by Gradle when it puts the docs folder in the jar.

Reply:
Why can't we just make the gradle task copy everything from `docs` to `test-runners/runner/plugins/Skript/doc-templates`?

Reply:
That's assuming we're not going to change the location, and also it can not be present, so it'll have to create the folders/files too. That is a solution.

If we put it in the jar, and make the map .txt solution, we could also use it for users that gen-docs with the commands